### DISCUSSION TITLE
Verbose progress of snapshot creation

### DISCUSSION THREAD

**Discussion by neothermic** [home-assistant/frontend#9109] [Supervisor]
Snapshots can take a very long time to create, especially full ones. It would help the impatience, or the mystery of whether it's simply stopped working greatly if the "Creating Snapshot" dialog box could show some progress, maybe which stage it's at and an estimate of the progression through the process.

**Comment by ludeeus** [comment]
This needs home-assistant/supervisor#1946 to be implemented first
